ajax/libs/v-mask/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "v-mask",
+  "filename": "v-mask.min.js",
+  "version": "1.3.2",
+  "description": "Simple input mask lib for vue.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/probil/v-mask.git"
+  },
+  "keywords": [
+    "vue.js",
+    "vue",
+    "vue-mask"
+  ],
+  "author": {
+    "name": "Max Lyashuk",
+    "url": "https://github.com/probil"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/probil/v-mask.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```

Comment:
Let's exclude `v-mask.esm.js` which is for esm. See https://github.com/probil/v-mask/blob/master/build/rollup.conf.js#L18-L21